client/src/pages/Product/Product.jsx

```diff
@@ -1,5 +1,101 @@
-import React from 'react';
+import React from "react";
+import {
+  Grid,
+  Typography,
+  Container,
+  IconButton,
+  Button,
+  Box,
+} from "@material-ui/core";
+import useStyles from "./styles";
+// import { useLocation } from "react-router";
+import Slider from "../../components/common/Slider/Slider";
+import GarantyIcon from "../../components/icons/GarantyIcon";
+import CheckCircleIcon from "../../components/icons/CheckCircleIcon";
 
-const Product = () => <div>Product page</div>;
+const phone = {
+  colors: ["white", "black", "#215787", "#C7F3BD", "#E70012"],
+  memory: [64, 128, 256],
+};
 
+const Product = () => {
+  const classes = useStyles();
+  //useLocation(id)
+
+  return (
+    <Container maxidth="xs">
```

Comment:
```suggestion
    <Container maxWidth="xs">
```
(Или я что-то не так понял)

Reply:
Это я видимо случайно( спасибо